docs/statistics.md

```diff
@@ -1,269 +1,425 @@
-to deliver (non-empty senders' buffer).
-Applicable for data sender.
+data to transmit, including packets that have been sent, but not yet acknowledged. In other words, the total accumulated duration in microseconds when there was something to deliver (non-empty senders' buffer). Available for sender.
 
-## pktSndDropTotal
+### pktSndDropTotal
 
-The number of "too late to send" packets dropped by sender (refer to `TLPKTDROP`).
+The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
+
+TODO: Update what's below -> Maxim to check -> issue on Github - update RFC and here put the link
 
 The total delay before `TLPKTDROP` is triggered consists of the `SRTO_PEERLATENCY`, 
 plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms).
 The delay used is the timespan between the very first packet and the latest packet 
 in the sender's buffer.
 
+### pktRcvDropTotal
+
+TODO: 
 
-## pktRcvDropTotal
+- missing?
+- Revise the text
+- create ticket -> rename back SRTO_TSBPDDELAY -> Maxim to create
 
-The number of "too late to play" missing packets. Receiver only.
+The total number of "too late to deliver" missing packets. Available for receiver.
 
-TSBPD and TLPacket drop socket option should be enabled.
+Missing packets here means lost or not yet received out-of-order packets. The receiver drops only those packets that are missing by the time there is at least one packet ready to be delivered to the upstream application.
 
-The receiver drops only those packets that are missing by the time there is at 
-least one packet ready to play.
+Also includes packets that failed to be decrypted (see [pktRcvUndecryptTotal](#pktRcvUndecryptTotal)). These 
+packets are present in the receiver's buffer and not dropped at the moment the decryption has failed. 
 
-Also includes packets that failed to be decrypted (pktRcvUndecryptTotal). These 
-packets are present in the receiver's buffer, and not dropped directly. 
+Note that `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` socket options should be enabled (refer to API.md - link!!!!).
 
-## pktRcvUndecryptTotal
+### pktRcvUndecryptTotal
 
-The number of packets that failed to be decrypted. Receiver side.
+The total number of packets that failed to be decrypted at the receiver side. Available for receiver.
 
-## pktSndFilterExtraTotal
+TODO: Is it correct that we include this number in the other statistics?
 
-The number of control packets supplied by the packet filter (refer to
+### pktSndFilterExtraTotal
+
+TODO: 
+
+- Packet filter control packets are SRT DATA packets. -> put this for all related stats
+- put this in FEC docs
+
+The total number of packet filter control packets supplied by the packet filter (refer to
 [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Sender only.
 
-Introduced in v1.4.0.
+Packet filter control packets are SRT DATA packets.
```

Comment:
Sure, but for statistics this doesn't mean anything. This packet is not calculated as sent data packets, nor is it treated as data packet at the reception side. I'm not sure (need to dig) how this packet is treated in case when it was detected as loss (the information about lost FEC control packets is available during group decommissioning in FEC, but I don't remember if this information is remembered anyhow in the stats).

Reply:
BTW. it would be worth checking if the FEC control packets are correctly calculated as the number of sent bytes, same as ACKACK packets, which maybe are small, but they do contribute to the overall bandwidth usage. Stating that packets sent and received do not count FEC control packets, the meaning of these fields is slightly changed. Maybe it's right to calculate them separately, but then the byte stats for FEC control packets should be also added.

Reply:
See issue #1211.